src/fleet/azext_fleet/custom.py

```diff
@@ -717,3 +775,278 @@ def approve_gate(cmd,  # pylint: disable=unused-argument
                  gate_name,
                  no_wait=False):
     return _patch_gate(cmd, client, resource_group_name, fleet_name, gate_name, "Completed", no_wait)
+
+
+def create_managed_namespace(cmd,
+                             client,
+                             resource_group_name,
+                             fleet_name,
+                             managed_namespace_name,
+                             labels=None,
+                             annotations=None,
+                             cpu_requests=None,
+                             cpu_limits=None,
+                             memory_requests=None,
+                             memory_limits=None,
+                             ingress_policy=None,
+                             egress_policy=None,
+                             delete_policy=None,
+                             adoption_policy=None,
+                             member_cluster_names=None):
+    managed_namespace_model = cmd.get_models(
+        "FleetManagedNamespace",
+        resource_type=CUSTOM_MGMT_FLEET
+    )
+
+    managed_namespace_properties_model = cmd.get_models(
+        "ManagedNamespaceProperties",
+        resource_type=CUSTOM_MGMT_FLEET
+    )
+
+    fleet_managed_namespace_properties_model = cmd.get_models(
+        "FleetManagedNamespaceProperties",
+        resource_type=CUSTOM_MGMT_FLEET
+    )
+
+    resource_limits = {}
+    if cpu_requests or cpu_limits or memory_requests or memory_limits:
+        if cpu_requests or memory_requests:
+            resource_limits['requests'] = {}
+            if cpu_requests:
+                resource_limits['requests']['cpu'] = cpu_requests
```

Comment:
maybe simplifiable a bit, e.g.

```suggestion
    resource_limits = {'requests':{}} # etc
    if cpu_requests:
        resource_limits['requests']['cpu'] = cpu_requests
```